transform-plugins/docs/XMLParser-transform.md

```diff
@@ -0,0 +1,82 @@
+Example
+-------
+
+This example parses the xml record received in "body" field of the input record, according to the
+xpathMappings specified, for each field name.
+The type output schema will be created, using the type specified for each field in "fieldTypeMapping".
+
+       {
+            "name": "XMLParser",
+            "plugin": {
+                "name": "XMLParser",
+                "type": "transform",
+                "label": "XMLParser",
+                "properties": {
+                    "encoding": "UTF-8",
+                     "processOnError": "Ignore error and continue",
+                      "xpathMappings": "category://book/@category,
+                                        title://book/title,
+                                        year:/bookstore/book[price>35.00]/year,
+                                        price:/bookstore/book[price>35.00]/price,
+                                        subcategory://book/subcategory",
+                      "fieldTypeMapping": "category:string,title:string,year:int,price:double,subcategory:string",
+                      "input": "body"
+                }
+            }
+       }
+
+For example, suppose the transform recieves the input record:
+
+    +=========================================================================================================+
+    | offset   | body                                                                                         |
+    +=========================================================================================================+
+    | 1        | <bookstore><book category="cooking"><subcategory><type>Continental</type></subcategory>      |
+    |          | <title lang=\"en\">Everyday Italian</title><author>Giada De Laurentiis</author><year>2005    |
+    |          | </year><price>30.00</price></book></bookstore>                                               |
+    | 2        | <bookstore><book category="children"><subcategory><type>Series</type></subcategory>          |
+    |          | <title lang=\"en\">Harry Potter</title><author>J K. Rowling</author><year>2005</year><price> |
+    |          | 49.99</price></book></bookstore>                                                             |
+    +=========================================================================================================+
+
+Output record will contain all the output fields specified by user:
+
+    +=========================================================================================================+
+    | category  | title              | year   |  price  | subcategory                                         |
+    +=========================================================================================================+
+    | cooking   | Everyday Italian   | null   |  null   | <subcategory><type>Continental</type></subcategory> |
+    | children  | Harry Potter       | 2005   | 49.99   | <subcategory><type>Series</type></subcategory>      |
```

Comment:
Would it be possible to include the multiple XML events case in the above example?